learn/wavemaker-release-notes/v10-5-6.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "WaveMaker 10.5.6 - Release date: 08 October 2020"
+id: ""
+sidebar_label: "Release v10.5.6"
+---
+*WaveMaker released a new update version with bug fixes.*
+
+## Improvements
+
+---
+
+- Stackable multiple toaster messages. This offers a way to display multiple toasters where you can configure the time duration.
```

Comment:
"Stackable multiple toaster messages. This offers a way to display multiple toasters in a page and can be configured via a simple method." We also need to create a how to doc for it.